salt/edx/templates/ansible_env_config.yml.j2

```diff
@@ -115,8 +115,8 @@ lms_auth_config:  &lms_auth
       password: "{{ MONGODB_CONTENTSTORE_PASSWORD }}"
       port: "{{ MONGODB_PORT }}"
       user: "{{ MONGODB_CONTENTSTORE_USER }}"
-    ADDITIONAL_OPTIONS:
```

Comment:
When I was looking through it appears that it gets that from the OPTIONS settings. I didn't find a reference to ADDITIONAL_OPTIONS outside of one location in the config repo, though I could have missed something.

My thinking is that we can try it with just putting the settings in the `OPTIONS` block and if it doesn't pick it up then we can add back in the `ADDITIONAL_OPTIONS`.